### ISSUE TITLE
Individual Public Activity not intuitively locateable, if still visible anywhere.

### ISSUE THREAD

**Issue by jonobrien** [isaacs/github#770]
It doesn't seem possible to see your most recent/current commits. There is no longer the `public activity` tab in your profile and I can not seem to see activity I just performed, commits and PRs not displaying as contributions on my profile page.

There is past events showing just fine, but up to the minute personal items are not displaying.

**Comment by aspiers** [comment]
Yeah, this is a really annoying regression.  To some extent it can be recovered via:

- https://api.github.com/users/$USERNAME/events

but of course this is barely human-readable.

**Comment by aspiers** [comment]
BTW it's not just commits, it's also comments on PRs and other activity.

**Comment by aspiers** [comment]
I've just discovered [`git hub`](http://pythonhosted.org/hub/) which is awesome and does this!

    $ git hub log
